templates/station-deployment.yaml

```
@@ -1,30 +1,52 @@
 {{- if .Values.station.enabled }}
+{{- $name := include "smartface.station.name" . -}}
+{{- $selectorLabels := include "smartface.station.selectorLabels" . -}}
 apiVersion: "apps/v1"
 kind: "Deployment"
 metadata:
-  name: {{ .Values.station.name | quote }}
+  name: {{ $name | quote }}
   labels:
-    app: {{ .Values.station.name | quote }}
+    {{- include "smartface.api.labels" . | nindent 4 }}
```

I believe there's a wrong value referenced here
```suggestion
    {{- include "smartface.station.labels" . | nindent 4 }}
```

---

thank you, fixed